localstack/utils/cloudformation/template_deployer.py

```diff
@@ -48,6 +48,7 @@
 REGEX_OUTPUT_APIGATEWAY = re.compile(
     rf"^(https?://.+\.execute-api\.)(?:[^-]+-){{2,3}}\d\.(amazonaws\.com|{AWS_URL_SUFFIX})/?(.*)$"
 )
+REGEX_DYNAMIC_REF = re.compile("{{resolve:([^:]+):(.+)}}")
```

Comment:
Just did a quick test against AWS, seems like it's possible to have multiple dynamic references in a single string, e.g. something like this:
```
Properties:
      TopicName: 't-{{resolve:secretsmanager:sec1}}-{{resolve:secretsmanager:sec2}}'
```
Not critical for now, but perhaps we can pick it up in the next iteration 👍